src/components/Fullscrn_qrcode.vue

```diff
@@ -50,16 +53,17 @@ img {
 .card-body {
   padding: 0.5rem;
 }
+.box {
+  text-align: center;
+}
 
 .bubble-light {
-  background-color: rgb(235, 235, 235);
-  border-color: rgb(235, 235, 235);
-  opacity: 0.8;
+  background-color: rgb(235, 235, 235, 0.75);

```

Comment:
You probably mean `rgba()`. Same goes for the following lines.

Reply:
fixed with 06650cb563424e310cfa3d362f0139ca70253197.